### PR TITLE
feat: list of groups in the vault

### PR DESCRIPTION
**Pull request by djaiss**

### CODE REVIEW

File: domains/Vault/ManageVault/Web/ViewHelpers/VaultShowViewHelper.php [hunk 1]
```diff
@@ -79,6 +76,12 @@ public static function upcomingReminders(Vault $vault, User $user): array
             ];
         });
 
+        // this line removes the null values that are added when the contact
+        // is not in the vault (in the method above)
+        $remindersCollection = $remindersCollection->filter(function ($value, $key) {
+            return $value != null;
+        });
+
```
Comment (djaiss) [inline domains/Vault/ManageVault/Web/ViewHelpers/VaultShowViewHelper.php:82]: @asbiin This is what I've added to fix the bug we had on reminders.

The `remindersCollection` might contain an empty item if the contact is not in the vault (in the method, we say `if contact->vault_id != vault, then return null`.
That creates an entry in the collection with `[ 0 => null]` in that case.
So the view didn't work anymore.
To solve this we need to remove all the entries in the collection that contains null entries.

Comment (asbiin) [reply domains/Vault/ManageVault/Web/ViewHelpers/VaultShowViewHelper.php:82]: And you can write:
```php
$remindersCollection = $remindersCollection->filter(fn ($value) => $value != null);
```

File: vault_id [hunk 2]
```diff
@@ -0,0 +1,3 @@
+PHPUnit 9.5.24 [44;37m#StandWith[0m[43;30mUkraine[0m
+
+No tests executed!
```
Comment (asbiin) [inline vault_id]: Is this file expected here?

**Review by asbiin — APPROVED** [review]
2 comments, but apart from that it's ok!

File: vault_id (hunk 2 above)
Comment (djaiss) [reply vault_id]: Not at all :-( Don't know what happened there.

**Pull request merged**